### PR TITLE
initial alpha version

### PR DESCRIPTION
**Pull request by kossel**
yup

### CODE REVIEW

**Comment by juankuquintana** [comment]
Please provide correct comments showing description of the work done

File: package.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  "keywords": [
+    "paybook",
+    "reactjs"
+  ],
+  "author": "Yichao",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/kossel/paybook-reactjs/issues"
+  },
+  "homepage": "https://github.com/kossel/paybook-reactjs#readme",
```
Comment (juankuquintana) [inline package.json]: no no

**Review by juankuquintana — DISMISSED** [review]

**Review by javikin — DISMISSED** [review]
Push commit to a new branch for a review

**Comment by kossel** [comment]
it's a forked project... not need branch

**Pull request merged**